bin/summariser.py

```diff
@@ -171,7 +172,9 @@ def runprocess(db_config_file, config_file):
             log.warning("The summariser may not start again until it is removed.")
 
         log.info(LOG_BREAK)
-
+        db.db.close()
+        query = ('DROP DATABASE ' + db_name)
+        subprocess.call(['mysql', '-u', 'root', '-e', query])
```

Comment:
Noooo! We really don't want to drop the database - this is potentially a production one we're connecting to!